R/qenv-replace_code.R

```diff
@@ -0,0 +1,53 @@
+#' Replace code
+#'
+#' Replace last element of code stored in `qenv`.
+#'
+#' Replace the last element of the character vector stored in `@code` with provided `code`.
+#' This can be used to mask sensitive information, e.g. credentials.
+#'
+#' @param object `qenv`
+#' @param code (`character` or `language`) code to set
+#'
+#' @return A modified `qenv`.
+#'
+#' @seealso [`qenv-class`], [`eval_code`]
+#'
+#' @examples
+#' q <- new_qenv()
+#' q <- eval_code(q, expression(i <- iris, ii <- subset(i, Species == "virginica")))
+#' get_code(q)
+#' q <- replace_code(q, expression(i <- iris, ii <- subset(i, Species != "virginica")))
```

Comment:
Replacing code with the same code? What do you think about this?
```suggestion
#' q <- eval_code(q, expression(print("secretpassword"))
#' get_code(q)
#' q <- replace_code(q, expression(print(askpass::askpass()))
```

Reply:
It wasn't the same though.